supabase/tests/66_test_on_user_org_access_queue.sql

```
@@ -125,9 +125,10 @@ SELECT is(
         "webhook_delivery",
         "credit_usage_posthog",
         "on_user_org_access",
-        "canceled_org_retention_alerts"
+        "canceled_org_retention_alerts",
+        "send_email"
     ]'::jsonb,
-    'high-frequency queues retain order and append on_user_org_access then canceled_org_retention_alerts'
+    'high-frequency queues retain order and append on_user_org_access, canceled_org_retention_alerts, then send_email'
```

SQLFluff (4.2.2)

[error] Line is too long (118 > 80).

(LT05)